specs-go/config.go

```diff
@@ -94,8 +94,12 @@ type Process struct {
 	SelinuxLabel string `json:"selinuxLabel,omitempty" platform:"linux"`
 	// IOPriority contains the I/O priority settings for the cgroup.
 	IOPriority *LinuxIOPriority `json:"ioPriority,omitempty" platform:"linux"`
+	// ExecCPUAffinity is Deprecated.
```

Comment:
If you want various linters to warn about using deprecated functionality (and other linters, that insist that description starts with the `Id`, to keep working fine), the deprecation notice should be added like this:

```go
// ExecCPUAffinity specifies CPU affinity for exec processes.
//
// Deprecated: use [CPUAffinity] instead.
```

(I.e. after the description, on a separate paragraph, etc.)

Reply:
Updated, similarly to how deprecation comment written in `Hooks` section